server/apps/macros/macro_register.py

```diff
@@ -44,7 +44,6 @@ def register_macros():
                      'callback' in dir(sys.modules[m]) and
                      not m.endswith('_test')
                      and not m.startswith('__')]
-    print(macro_modules)
```

Comment:
There was a hack introduced long time back to solve the problem where macros were not getting loaded for celery jobs. I am not sure if `print` was the hack or not. But it would be worth checking if the macro functionality works fine with celery jobs or not?